charts/prometheus/Chart.yaml

```
@@ -26,6 +26,6 @@ engine: gotpl
 type: application
 dependencies:
   - name: kube-state-metrics
-    version: "2.13.*"
+    version: "3.0.*"
     repository: https://kubernetes.github.io/kube-state-metrics
```

repository needs to be changed: 
```suggestion
    repository: https://prometheus-community.github.io/helm-charts
```